src/routes/jira/jira-atlassian-connect-get.ts

```diff
@@ -87,6 +87,14 @@ const modules = {
 			},
 			url: "/jira/app-creation",
 			conditions: adminCondition
+		},
+		{
+			key: "github-manual-app-creation-page",
+			name: {
+				value: "GitHub Manual App Creation"
+			},
+			url: "/jira/manual-app-creation/?gitHubServerAppId={ac.ghsaId}",
```

Comment:
I'm not sure the id is needed at this stage...

Reply:
The id is needed here to fetch the `gitHubBaseUrl` and `uuid` of the `GithubServerApp` in order to display the _callback_, _webhook_ and _setup_ URLs in the manual app creation page.

Reply:
Does anyone know the need of 
```
{
			key: "github-manual-app-creation-page",
			name: {
```
Is it more a UX requirement (so that user can jump to here directly) ? Or is it a tech requirement?
Since we are iframing our app, doesn't the `window.location.url=xxx` or `history.back()` work without above?

Reply:
We shouldn't be creating the module for the manual app creation screen as part of this PR. Plus, that path wouldn't need an id as it's simply a form that will render a couple of urls and take in user inputs.

Reply:
See this recording for the issue we're trying to fix.

https://user-images.githubusercontent.com/37155488/178183440-efd650c0-b059-4431-b766-e65c9439bd94.mov

In the cloud flow, after I hit connect, instead of being redirected to the GH config page, I land on the select GH version screen because we're simply calling `window.close()`. This worked when we only had one screen on the Jira side but obviously needs to be modified to take the user back to the correct place. 

All the manual app creation changes are not needed to resolve this.

Reply:
This has been reverted.